proposals/vector_to_points.json

```diff
@@ -0,0 +1,68 @@
+{
+    "id": "vector_to_points",
+    "summary": "Sample random points from geometries",
+    "description": "For each geometry samples a number of random points based on one of the sampling method.",
```

Comment:
> based on one of the sampling method.

typo, I guess must be

> based on a sampling method

Reply:
Yeah, we could either append an "s" or use your suggestion

```suggestion
    "description": "For each geometry samples a number of random points based on a sampling method.",
```

Reply:
To be honest, "sample based on a sampling method"  sounds quite redundant to me.

If you want the description different enough from the summary, something like this:

> Generate a vector cube of points by sampling random points from input geometries.

Reply:
Works for me except that "random" would sound a bit like just the "random" method is available although we (at least currently) still have "fibonacci" available, too.

Reply:
yeah, the random can be dropped

Reply:
```suggestion
    "description": "Generate a vector cube of points by sampling points from input geometries.",
```

I hope this is long enough...